features/storage/kvstore/conf/tdb_internal/mbed_lib.json

```diff
@@ -15,6 +15,10 @@
             "internal_size": "0x8000",
             "internal_base_address": "0x00028000"
         },
+        "DISCO_H747": {
```

Comment:
Wrong target name :-(

Reply:
Darn diddly, good catch. It's fixed now.